modules/azure/src/main/java/org/testcontainers/containers/CosmosDBEmulatorContainer.java

```diff
@@ -0,0 +1,99 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import static org.testcontainers.containers.Constants.EMULATOR_CERTIFICATE_FILE_NAME;
+import static org.testcontainers.containers.Constants.EMULATOR_KEY;
+import static org.testcontainers.containers.Constants.KEYSTORE_FILE_NAME;
+import static org.testcontainers.containers.Constants.STORE_PASSWORD;
+import static org.testcontainers.containers.Constants.STORE_TYPE;
+import static org.testcontainers.containers.Constants.TEMP_DIRECTORY_NAME;
+import static org.testcontainers.utility.DockerImageName.parse;
+
+/**
+ * An Azure CosmosDB container
+ *
+ * Default port is 8081.
+ *
+ * @author Onur Kagan Ozcan
+ */
+public class CosmosDBEmulatorContainer extends GenericContainer<CosmosDBEmulatorContainer> {
+
+    public static final String LINUX_AZURE_COSMOS_DB_EMULATOR = "mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator";
```

Comment:
Is there a reason this should be `public`? Else, it's also fine to inline it in `parse()`.

Reply:
Inconvenient as it is, this should not be public. Having image names in public constants (or default constructors) is something we're trying to remove from Testcontainers. Users should provide an image name themselves.

Reply:
👍 Added to my patch list. Actually I did this by looking into other implementations. If design choose was this, than will be removed.